src/Command.js

```diff
@@ -182,6 +183,13 @@ export default class Command {
       if (this.getOptions().includeFilteredDependencies) {
         this.filteredPackages = PackageUtilities.addDependencies(this.filteredPackages, this.packageGraph);
       }
+      if (this.flags.onlyUpdated) {
```

Comment:
Looks like it'll also work for `clean`, `ls` and `bootstrap` however I can move the logic to a utility fn and call it separately in `exec` and `run` if you prefer.